website/docs/error-reference.md

```diff
@@ -388,6 +388,20 @@ See [5014](#5014). 5036 is the same error as [5014](#5014) but slightly modified
 to allow more common Ruby idioms to pass by in `# typed: true` (5036 is only
 reported in `# typed: strict`).
 
+## 5041
+
+Sorbet does not allow inheriting from a class which inherits from `T::Struct`.
```

Comment:
Do you think we should explain why we impose this limitation?

Reply:
cc @jvilk-stripe the ticket was asking why.

Reply:
> Do you think we should explain why we impose this limitation?

Yes, I think we should explicitly name why this limitation exists, and not document InexactStruct.